targets/targets.mk

```diff
@@ -32,4 +32,5 @@ $(eval $(call GluonTarget,bcm27xx,bcm2711)) # BROKEN: No 11s support, no reset b
 $(eval $(call GluonTarget,ipq40xx,chromium)) # BROKEN: Devices cannot be flashed without opening the case
 $(eval $(call GluonTarget,kirkwood,generic)) # BROKEN: No devices with 11s support
 $(eval $(call GluonTarget,mvebu,cortexa9)) # BROKEN: No 11s support
+$(eval $(call GluonTarget,mvebu,cortexa53)) # BROKEN: Wireless untested
```

Comment:
Since the device in question doesn't have Wireless capabilities and the checklist looks fine to me, I would argue the broken classification isn't necessary here.
The requirement is simply that if a device has those capabilities, then it has to work. That way no false expectations are created. However, if it doesn't, that is completely fine.

If it turns out that WLAN is a problem with that target it should probably be flagged for the device in question and not the whole target.

So if you would like to add it as non-broken just move it in the correct alpabetically place in the list above and then create the docs entry.

Reply:
why I think about it is because there is a same device where that has also wifi and it is the same image

the: GL-MV1000W

https://openwrt.org/toh/gl.inet/gl-mv1000

> The Brume-W, GL-MV1000W, is the same hardware plus a WLAN card, and will run on the same image. It contains an additional Wireless USB chip. Please see the section wireless for details.